src/backend/distributed/deparser/deparse.c

```diff
@@ -98,6 +120,14 @@ DeparseDropStmt(DropStmt *stmt)
 			return DeparseDropTypeStmt(stmt);
 		}
 
+		case OBJECT_FUNCTION:
+#if PG_VERSION_NUM >= 110000
```

Comment:
nit; if you move this block above `case OBJECT_FUNCTION` the indent of the body is a bit more consistent, there are a couple more in this PR.